### PR TITLE
Swagger Integration

### PR DESCRIPTION
**Pull request by ixidion**
Implementation of #1 
Swagger is directly integrated with JavaScript without the iFrame.
Swagger Dep is integrated with composer.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,11 +14,13 @@
...
         "laravel/tinker": "~1.0",
         "league/flysystem-aws-s3-v3": "^1.0",
-        "predis/predis": "^1.1"
+        "predis/predis": "^1.1",
+        "swagger-api/swagger-ui": "^3.2"
     },
     "require-dev": {
         "filp/whoops": "~2.0",
         "fzaninotto/faker": "~1.4",
+        "laravel/homestead": "^6.2",
```
Comment (poma) [inline composer.json]: do we really need to add this dependency?

**Comment by poma** [comment]
also need to remove swagger route `routes/web.php`

**Comment by poma** [comment]
Everything else checked, ready to merge

File: composer.json (hunk 1 above)
Comment (ixidion) [reply composer.json]: Can be removed. Is only needed, if you use homestead per project approach.

**Pull request merged**